AK/FlyString.h

```diff
@@ -21,6 +21,10 @@ class FlyString {
     ~FlyString();
 
     static ErrorOr<FlyString> from_utf8(StringView);
+    template<typename T>
+    requires(IsOneOf<RemoveCVReference<T>, DeprecatedString, DeprecatedFlyString>)
```

Comment:
```suggestion
    template<OneOfIgnoringCV<DeprecatedString, DeprecatedFlyString> T>
```

(plus others in this commit)

Reply:
oh I just noticed that this also strips references, nevermind.